Makefile

```diff
@@ -81,8 +81,8 @@ golint:
 
 govet:
 	@echo "Running go vet"
-	# Disabling GO111MODULE just for go vet execution
-	GO111MODULE=off go vet github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites...
+	cnf-tests/hack/govet.sh
+	ztp/ztp-policy-generator/kustomize/plugin/policyGenerator/v1/policygenerator/hack/govet.sh
```

Comment:
why do you hide the ztp scripts dir so deep?

Reply:
or to that matter the whole policyGenerator

Reply:
weirdly, the previous statement here doesn't seems to work (weird, because I'd expect that to cause a CI error)

Reply:
Not sure.  But I'm not going to change that today :)

Reply:
My guess is that for kustomize to find the plugin the executable has to be in the proper directory structure.  Maybe we could move the source out to a more manageable depth and just have a makefile that copies the binary deep into that tree?

@serngawy can you shed some light on this?  Does the source need to be that deep, or just the resulting executable?

Reply:
for the kustomize plugin it needs to follow this directory structure to follow kustomize documentation 
https://kubectl.docs.kubernetes.io/guides/extending_kustomize/

Reply:
@serngawy : But all the source doesn't need to be there, right?  If we had the source in, say `ztp/PolicyGenerator` with a makefile that copied the executable into `ztp/ztp-policy-generator/kustomize/plugin/policyGenerator/v1/policygenerator` that would work too, right?

Reply:
yes, the binary that matter.

Reply:
@serngawy  Is there a list of "We should do this some day" cleanup stuff for the policygenerator?  I'm not going to do that in this PR, obviously.